.travis.yml

```diff
@@ -195,7 +195,7 @@ install:
             LIBDPDKEXT=a
            fi
            DPDK_TARGET="${DPDK_TARGET}gcc"
-           if [ ! -f "dpdk/${TARGET}/lib/libdpdk.$LIBDPDKEXT" ]; then
+           if [ ! -f "dpdk/${TARGET}/usr/local/lib/libdpdk.$LIBDPDKEXT" ]; then
```

Comment:
that should not be correct.  dpdk is installed to $HOME/dpdk/ dir. And there lib/ and etc folders are created. Also dpdk directory is cached by travis to not rebuild it each time.

Reply:
This shouldn't change anything. DPDK will still be installed in the home directory (DESTDIR is still $HOME/dpdk/$TARGET), the difference is that now you will be testing with an installed version of DPDK rather than the directory with build artifacts.

Maybe we should set "prefix" when installing to something more meaningful like dpdk-17.08 instead of the default /usr/local, and also set DESTDIR to $HOME. This way the check done earlier can just look for the presence of this directory rather than grepping a file inside the build directory. This could also allow us to have several versions of DPDK installed which we could test ODP against. If disk space in Travis is not an issue, that is.